### DISCUSSION TITLE
Foreign key support for views

### DISCUSSION THREAD

**Discussion by sparkofreason** [tamizhvendan/honeyeql#9] [Ideas]
Currently honeyeql does not provide the ability to navigate from views via foreign keys defined on the referenced table(s); at least this is the case in MySQL. This actually appears to be an issue from JDBC which does not return foreign keys on views. I haven't been able to find a very good way of implementing this short of parsing the SQL which defines the view, but thought I'd open it up to discussion if anyone has any ideas here.

**Comment by tamizhvendan** [comment]
@sparkofreason I plan to address this scenario by explicitly passing the relationship during the initialization. I am yet to iron out the details. The challenge is to find an uninterrupted hours of time :(

**Reply by sparkofreason** [reply]
Thanks for the update. Let me know if there's something I can do to help. I have spent some time in the code.
